src/esp/physics/bullet/BulletRigidScene.h

```diff
@@ -14,6 +14,12 @@
 namespace esp {
 namespace physics {
 
+/**
+ * @brief An individual rigid scene instance implementing an interface with
+ * Bullet physics to enable dynamics. See @ref btCollisionObject for @ref
+ * RigidObjectType::SCENE
```

Comment:
~RigidObjectType~ We should clean-up any other references to this. I pointed these out when I saw them.